packages/mathtype-html-integration-devkit/src/integrationmodel.js

```diff
@@ -387,6 +387,14 @@ export default class IntegrationModel {
     );
   }
 
+  /**
+   * Remove events to formulas in the DOM target.
```

Comment:
Does it mean we remove all the events created by mathtype for the formulas?

Reply:
That's it. This function, it is called when CK5 is destroyed, and remove:

- dblClick, mouseDown, mouseUp event from eventTarget.
- mouseUp event from document.

These four events are added each time that CK5 with MathType is created. If we don't remove them each time that CK5 it is destroyed, the garbage collector will not destroy the CKEditor5Integration object.